### DISCUSSION TITLE
Kunitsu-Gami: Path of the Goddess

### DISCUSSION THREAD

**Discussion by LittleTale** [praydog/REFramework#1087] [General Questions] [answered]
Quick question, does the nightly built work with the  Kunitsu-Gami: Path of the Goddess DEMO ? game doesn't have ultra wide screen supports sadly...
Thanks in advance for your kind answer.

**Comment by praydog** [comment, marked as answer]
Yes, use the DD2 build, it works fine. If you want to use ultrawide, make sure to enable the "Ultrawide: Override FOV" option under it and keep it at 1.0.

EDIT: Use the nightly build for latest fixes and stability changes, and black bar removals: https://github.com/praydog/REFramework-nightly/releases/

**Reply by LittleTale** [reply]
Thanks a lot for the quick response !
